### ISSUE TITLE
-80

### ISSUE THREAD

**Issue by FazziCLAY** [hollow-cube/minestom-ce#25]
```java
org.jglrxavpok.hephaistos.mca.AnvilException: Trying to load chunk with minY = -80, but instance dimension type (minecraft:overworld) has a minY of -64
	at net.minestom.server.instance.AnvilLoader.loadMCA(AnvilLoader.java:109)
	at net.minestom.server.instance.AnvilLoader.loadChunk(AnvilLoader.java:87)
	at net.minestom.server.instance.InstanceContainer.lambda$retrieveChunk$8(InstanceContainer.java:266)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
```

**Comment by mworzala** [comment]
Seems like you just need to change the dimension type to have a lower minimum if that's what your world is

**Comment by TheMeinerLP** [comment]
We fixed it in our fork.
You need to remove in the Anvil loader following lines: https://github.com/hollow-cube/minestom-ce/blob/74ca1041f3ddd9f58f89583a611644fb23ed7d16/src/main/java/net/minestom/server/instance/AnvilLoader.java#L106-L122

**Comment by mworzala** [comment]
What... that is not a fix, you just need to configure your dimension height correctly

**Comment by FazziCLAY** [comment]
> 

I got this error when I just tried to download a world created a minute ago by minecraft.

**Comment by ZakShearman** [comment]
> > 
> 
> I got this error when I just tried to download a world created a minute ago by minecraft.

And have you configure the dimension type's minY value?

**Comment by TheMeinerLP** [comment]
> What... that is not a fix, you just need to configure your dimension height correctly

You can't the code is here wrong based of the newer World System

**Comment by mworzala** [comment]
Deleting those checks is definitely not the correct solution. I could certainly believe that there is a bug, but then some changes are needed. Not deleting those lines completely.

**Comment by mworzala** [comment]
@FazziCLAY will you share the world file?

**Comment by BomBardyGamer** [comment]
> > What... that is not a fix, you just need to configure your dimension height correctly
> 
> You can't the code is here wrong based of the newer World System

Why are you insistent that just removing the perfectly valid and reasonable code that validates that chunks with values that don't match up with the dimension type is a good solution?

**Comment by mworzala** [comment]
Similar issue report from minestom https://discord.com/channels/706185253441634317/706186227493109860/1134534839354277908

**Comment by FazziCLAY** [comment]
> @FazziCLAY will you share the world file?

I deleted that world, but recreating an exact copy is very easy, here are the steps:
- Open minecraft 1.20.1
- create a new world (super-flat, the_void)
- done..

**Comment by TheMeinerLP** [comment]
I tested also little around. Maybe interesting, my 1.19.4 world are loading fine. 
Also i got the information highest point and lowest point are written in the data packs from vanilla. That's can be extracted with the generator.

**Comment by Eoghanmc22** [comment]
Should be fixed in Hephaistos 2.6.1. See jglrxavpok/Hephaistos#35
